zcash_client_backend/src/data_api.rs

```diff
@@ -809,6 +809,15 @@ pub trait InputSource {
     ) -> Result<Vec<WalletTransparentOutput>, Self::Error> {
         Ok(vec![])
     }
+
+    /// Return the notes with a given value
+    #[cfg(any(test, feature = "test-dependencies"))]
+    fn get_notes(
```

Comment:
In #1539 I'm moving these to a dedicated `WalletTest` trait so we can freely expose whatever internals are needed for testing purposes. (I'll handle any conflicts.)